Sources/KVKCalendar/TimelineView.swift

```diff
@@ -212,6 +212,17 @@ final class TimelineView: UIView, EventDateProtocol, CalendarTimer {
         scrollView.scrollRectToVisible(frame, animated: true)
     }
     
+    private func scrollToHour(_ hour: Int) {
+        guard let time = getTimelineLabel(hour: hour) else {
+            scrollView.setContentOffset(.zero, animated: true)
+            return
+        }
+        
+        var frame = scrollView.frame
+        frame.origin.y = time.frame.origin.y - 10
```

Comment:
Why `-10`? 🤔

Reply:
Because I was guided by a function similar in functionality with scrollToCurrentTime on line 201 in the TimelineView file